### ISSUE TITLE
[Feature request] Auto Reconnect After N Minutes (Set by user)

### ISSUE THREAD

**Issue by SolidAnonDev** [mullvad/mullvadvpn-app#7077]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

I often turn off my VPN for various reasons. Silly example - I often find myself disabling to quickly click a link on reddit, blocked by network security (due to being logged out in private mode and on VPN).

I then forget to turn the VPN back on. I'd like to see a feature where I can set something that automatically turns the VPN on after n number of minutes, for example, reconnect after one hour. A user could set this in their settings and their VPN would always be turning itself back on after their specified time frame, making it such that even if I forget to turn it back on, I am covered for the long term.

### Alternative solutions

Alternatively, possibly a reminder in the form of a push notification? This might at least allow for a user to be reminded that the VPN is off, again, something that might be able to be set in the settings.

### Type of feature

- [X] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [X] Android
- [X] iOS
- [X] Windows
- [X] macOS
- [X] Linux

**Comment by MrChocolatine** [comment]
This looks like #3652 , where instead of asking to auto-reconnect after X minutes it is asked to turn off the VPN for X minutes, but the goal is the same 👍 .

Answer from the team: https://github.com/mullvad/mullvadvpn-app/issues/3652#issuecomment-1154010162

**Comment by faern** [comment]
Closing as duplicate of #8909 (previously #3652)
